rllib/agents/trainer.py

```diff
@@ -37,6 +37,7 @@
 from ray.rllib.execution.train_ops import TrainOneStep, MultiGPUTrainOneStep, \
     train_one_step, multi_gpu_train_one_step
 from ray.rllib.models import MODEL_DEFAULTS
+from ray.rllib.offline import get_offline_io_resource_bundles
```

Comment:
Nit: Full path to imported module.

Reply:
sorry, I bit confused, this is the full path I thought?
can you clarify a bit? thanks.

Reply:
Sorry, I meant:
```
from ray.rllib.offline.resource import get_offline_io_resource_bundles
```